apm-sniffer/apm-agent-core/pom.xml

```diff
@@ -22,7 +22,7 @@
     <parent>
         <groupId>org.apache.skywalking</groupId>
         <artifactId>java-agent-sniffer</artifactId>
-        <version>9.4.0-SNAPSHOT</version>
+        <version>9.3.0-SNAPSHOT</version>
```

Comment:
I think you merged the things wrong. We are in 9.4.0 iteration. You reverted when merging.

Reply:
sorry, i neglected.